include/sample_rdd.h

```diff
@@ -0,0 +1,92 @@
+#ifndef CPARK_SAMPLE_RDD_H
+#define CPARK_SAMPLE_RDD_H
+
+#include <random>
+#include <vector>
+#include "base_rdd.h"
+#include "filter_rdd.h"
+#include "utils.h"
+namespace cpark {
+
```

Comment:
I would suggest adding the doxygen-required comments since the main documentation has already been deployed, later commited file should handle this before the merge. 
```suggestion

/** @ingroup g_trans
 *  @defgroup t_Sample Sample
 *  This forms the Sample Transformation of our cpark library
 *  @image html sample.drawio.png "SampleRdd diagram" width=50%
 *  @image latex sample.drawio.png "SampleRdd diagram" width=10cm
 *  @see SampleRdd
 *  @see Sample
 *  @{
 */
```

Reply:
Got it. Will refer to the existing main repo to add the comments

Reply:
Thanks for your reply. 
Yes, providing a more detailed text description is welcomed. 
However, to ease your work, I have already provided the most basic solution in the reviews. If you want, you can just click Commit Suggestion on GitHub.